yt_dlp/extractor/germanupa.py

```diff
@@ -0,0 +1,91 @@
+        'info_dict': {
+            'id': '1867346676',
+            'title': 'Live vom UX Festival: Neuigkeiten von Figma, Jobmarkt, Agenturszene & Interview zu Sustainable UX',
+            'ext': 'opus',
+            'timestamp': 1720545088,
+            'upload_date': '20240709',
+            'duration': 3910.557,
+            'like_count': int,
+            'description': 'md5:db2aed5ff131e177a7b33901e9a8db05',
+            'uploader': 'German UPA',
+            'repost_count': int,
+            'genres': ['Science'],
+            'license': 'all-rights-reserved',
+            'uploader_url': 'https://soundcloud.com/user-80097677',
+            'uploader_id': '471579486',
+            'view_count': int,
+            'comment_count': int,
+            'thumbnail': 'https://i1.sndcdn.com/artworks-oCti2e9GhaZFWBqY-48ybGw-original.jpg',
+        },
+    }, {
+        'note': 'Nur für Mitglieder/Just for members',
+        'url': 'https://germanupa.de/mediathek/ux-festival-2024-usability-tests-und-ai',
+        'info_dict': {
+            'id': '986994430',
+            'title': 'UX Festival 2024 "Usability Tests und AI" von Lennart Weber',
+            'ext': 'mp4',
+            'release_date': '20240719',
+            'uploader_url': 'https://vimeo.com/germanupa',
+            'timestamp': 1721373980,
+            'license': 'by-sa',
+            'like_count': int,
+            'thumbnail': 'https://i.vimeocdn.com/video/1904187064-2a672630c30f9ad787bd390bff3f51d7506a3e8416763ba6dbf465732b165c5c-d_1280',
+            'duration': 2146,
+            'release_timestamp': 1721373980,
+            'uploader': 'German UPA',
+            'uploader_id': 'germanupa',
+            'upload_date': '20240719',
+            'comment_count': int,
+        },
+        'expected_warnings': ['Failed to parse XML: not well-formed'],
+        'skip': 'login required',
+    }]
+    _IFRAME_RE = r'<iframe[^>]+data-src\s*?=\s*?([\'"])(?P<url>https://germanupa\.de/media/oembed\?url=(?:(?!\1).)+)\1'
+    _LOGIN_REQUIRED_RE = r'<div[^>]+class\s*?=\s*?([\'"])(?:(?!\1).)*login-wrapper(?:(?!\1).)*\1'
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        embed_url = self._search_regex(self._IFRAME_RE, webpage, 'embed video', None, group='url')
+        if not embed_url:
+            return self.url_result(url, 'Generic')  # Fall back to generic to extract audio
+
+        param_url = traverse_obj(parse_qs(embed_url), ('url', 0, {url_or_none}))
+        if not param_url:
+            if self._search_regex(self._LOGIN_REQUIRED_RE, webpage, 'login wrapper', default=None):
+                self.raise_login_required('This video is only available for members')
+            raise ExtractorError('Failed to extract URL from the query parameters', video_id=video_id)
```

Comment:
No need to pass the video id
```suggestion
            raise ExtractorError('Failed to extract URL from the query parameters')
```